RetailCoder.VBE/Inspections/ImplicitActiveSheetReferenceInspection.cs

```diff
@@ -39,8 +37,7 @@ public override IEnumerable<InspectionResultBase> GetInspectionResults()
             }
 
             var matches = BuiltInDeclarations.Where(item =>
-                (item.ParentScope == "Excel.Global" || item.ParentScope == "Excel.Application")
-                && Targets.Contains(item.IdentifierName)).ToList();
+                Targets.Contains(item.IdentifierName)).ToList();
 
             var issues = matches.Where(item => item.References.Any())
                 .SelectMany(declaration => declaration.References);
```

Comment:
Could you add that `Where` predicate into the `matches` query? Actually kill the `matches` query, and make `issues` read like it does in `ImplicitActiveWorkbookReferenceInspection`, just for consistency's sake :wink: